programs/locker/src/instructions/v3/close_claim_status.rs

```diff
@@ -0,0 +1,56 @@
+use crate::*;
+use anchor_lang::system_program;
+use util::{account_info_ref_lifetime_shortener, close};
+
+/// Accounts for [locker::close_claim_status].
+#[derive(Accounts)]
+#[event_cpi]
+pub struct CloseClaimStatus<'info> {
+    /// Claim status
+    #[account(
+        mut,
+        has_one = recipient,
+        has_one = escrow,
+        close = recipient,
+    )]
+    pub claim_status: Account<'info, ClaimStatus>,
+
+    /// CHECK: this account use to verify escrow cancelled or closed
+    pub escrow: UncheckedAccount<'info>,
+
+    /// recipient
+    #[account(mut)]
+    recipient: Signer<'info>,
```

Comment:
we should use rent_receiver to get rent fee

Reply:
added rent_receiver.